packages/md/package.json

```diff
@@ -17,13 +17,15 @@
 	"dependencies": {
 		"@flint.fyi/core": "workspace:",
 		"@flint.fyi/rule-tester": "workspace:",
+		"github-slugger": "^2.0.0",
```

Comment:
🤔 I would like to see if this is possible without a GitHub-specific library, or even without a dependency altogether. TBD.

Reply:
Marking as blocked pending more investigations.

Reply:
`github-slugger` is _the_ library that the Remark ecosystem uses (as well as `eslint-plugin-markdown-links`), and because we're already using Remark here, it's not actually adding a new dependency after deleting `@types/github-slugger` (`github-slugger` ships its own types nowadays).
If we want to support other platforms, a different library might make sense, but if we're only parsing GFM 99% of the time (which, excepting the Ruby GH itself, will likely use `github-slugger` themselves), I don't know it makes much sense to block this on it.

Reply:
Ok great that's the kind of investigation I was hoping for haha

Reply:
Good reads both! (sorry, got distracted reading the linked tseslint issues, yeah maybe I have a problem :))

(Also, tbh you've got more #s memorized than me. I don't do #s)

(Anyway, this is really off topic, 🤐)